### ISSUE TITLE
Changing the chain ID when running fork mainnet?

### ISSUE THREAD

**Issue by AoDev** [eth-brownie/brownie#901]
### Overview

I'd like to be able to change the chain id from 1337 to 1 when running 
`brownie console --network mainnet-fork`

In general, I don't see how to change ganache cli settings when running mainnet-fork.
I tried to set a config for 'mainnet-fork' under the networks config of `brownie-config.yaml` but didn't see the changes reflected.

**Comment by iamdefinitelyahuman** [comment]
Ganache exposes a `--chainId` flag to set the chain id at launch.  It should be pretty simple (and makes sense) to set this according to the forked chain.

**Comment by AoDev** [comment]
Current solution has been to modify `network-config.yaml` found at `~/.brownie` in this way:

```yaml
development:
  - name: Ganache-CLI (Mainnet Fork)
    id: mainnet-fork
    cmd: ganache-cli --chainId 1 --networkId 1 # <--- HERE
    host: http://127.0.0.1
    timeout: 120
    cmd_settings:
      port: 8545
      gas_limit: 12000000
      accounts: 10
      evm_version: istanbul
      mnemonic: brownie
      fork: mainnet
```

**Comment by iamdefinitelyahuman** [comment]
Implemented in #908
